buildSrc/src/main/kotlin/criterion/benchmark.kt

```diff
@@ -14,19 +14,25 @@
  * limitations under the License.
  */
 
+
 /**
  * A small, criterion-like benchmarking framework.
  */
+
```

Comment:
:x: It would be nice if we could avoid these two empty lines between the kdoc comment and its associated `package` declaration.